dashboard/src/components/js/Modal/Modal.js

```diff
@@ -46,13 +46,13 @@ const Modal = ({
                 type="button"
                 onClick={onClose}
               >
-                <clr-icon aria-hidden="true" shape="close" />
+                <cds-icon aria-hidden="true" shape="window-close" />
               </button>
             )}
             <h3 className="modal-title">{title}</h3>
           </div>
           <div className="modal-body">{children}</div>
-          <div className="modal-footer">{footer}</div>
+          {footer && <div className="modal-footer">{footer}</div>}
```

Comment:
Hide footer to prevent an extra space from being rendered when no footer is passed

Reply:
you can remove these changes now?

Reply:
Why? Do you mean removing all the changes related to the old modals? Or just this one?

Reply:
I mean the changes related to this file Modal.js, you are not using it in the PR, no?